frontend/src/Components/Map.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { tileNames, tilesMap } from "./tilesMap.js";
+import "./Map.css";
+
+class Map extends React.Component {
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  randomGrass() {
+    let randomNum = Math.floor(Math.random() * 40);
+    if (randomNum === 6 || randomNum === 23) {
+      return "highGrass";
+    } else if (randomNum === 21) {
+      return "flowers";
+    } else {
+      return "grass";
+    }
+  }
+
+  render() {
+    return (
+      <div className="map">
+        {tilesMap.map(row =>
+          row.map(column => {
```

Comment:
I'm not working with a whole "column" in this case, but with one "tile" ("one column in a one-row-array", if you prefer !)